x-pack/legacy/plugins/canvas/i18n/components.ts

```diff
@@ -9,17 +9,17 @@ import { CANVAS, JSON, KIBANA, PDF, POST, URL } from './constants';
 
 export const ComponentStrings = {
   App: {
-    getLoadErrorTitle: () =>
-      i18n.translate('xpack.canvas.app.loadErrorTitle', {
-        defaultMessage: 'Canvas failed to load :(',
-      }),
     getLoadErrorMessage: (error: string) =>
       i18n.translate('xpack.canvas.app.loadErrorMessage', {
         defaultMessage: 'Message: {error}',
         values: {
           error,
         },
       }),
+    getLoadErrorTitle: () =>
+      i18n.translate('xpack.canvas.app.loadErrorTitle', {
+        defaultMessage: 'Canvas failed to load :(',
```

Comment:
Should we take out the text emoji from the string `:(`? Will it be confusing to translators?